pygsti/objectivefns/objectivefns.py

```diff
@@ -29,6 +31,11 @@
 from pygsti.models.model import OpModel as _OpModel
 from pygsti import SpaceT
 
+
+DEFAULT_MIN_PROB_CLIP = 1e-4
+DEFAULT_RADIUS        = 1e-4
```

Comment:
Should these be class attributes of the base `ObjectiveFunction` class instead of at the module level? (I don't feel strongly either way.)

Reply:
I have a small preference for defining these at the module level. The inheritance structure of classes in objectivefns.py is already very complicated, and I'd like to avoid adding complexity to the base classes. That sound okay?

Reply:
For posterity:

Corey [replied](https://github.com/sandialabs/pyGSTi/pull/671#issuecomment-3456857509) "Aye."

Marking as resolved.